deploy/docker/pump_wrapper.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
```

Comment:
This is super hacky. We should just fix the underlying deficiency, which is #1202. I'll try to do that soon.